config.js

```diff
@@ -0,0 +1 @@
+export const basename = '/isearch/0.14/index.html';
```

Comment:
Ah yes could do that. I have added a line in the gulpfile which does something similar but your suggestion might be simpler!